modules/vaos/spec/factories/ccra/referrals.rb

```diff
@@ -21,20 +21,48 @@
   factory :ccra_referral_detail, class: 'Ccra::ReferralDetail' do
     category_of_care { 'CARDIOLOGY' }
     provider_name { 'Dr. Smith' }
-    location { 'VA Medical Center' }
+    provider_npi { '1234567890' }
+    provider_telephone { '555-987-6543' }
+    treating_facility { 'VA Medical Center' }
     referral_number { 'VA0000005681' }
-    expiration_date { '2024-05-27' }
-    treating_facility_phone { '555-123-4567' }
+    expiration_date { (Date.current + 30.days).to_s }
+    phone_number { '555-123-4567' }
+    referring_facility_name { 'Dayton VA Medical Center' }
+    referring_facility_phone { '(937) 262-3800' }
+    referring_facility_code { '552' }
+    referring_facility_address1 { '4100 West Third Street' }
+    referring_facility_city { 'DAYTON' }
+    referring_facility_state { 'OH' }
+    referring_facility_zip { '45428' }
+    has_appointments { 'Y' }
 
     initialize_with do
       attributes = {
         'Referral' => {
           'CategoryOfCare' => category_of_care,
-          'TreatingProvider' => provider_name,
-          'TreatingFacility' => location,
+          'TreatingFacility' => treating_facility,
```

Comment:
Note: The property names started with lowercase in the old version, while the CCRA spec calls for starting with uppercase. I don't think that is correct on their side, but that is what they are providing.